feature-libs/cart/main/core/facade/active-cart.service.spec.ts

```diff
@@ -217,6 +217,16 @@ describe('ActiveCartService', () => {
         });
     });
 
+    describe('clearActiveCart', () => {
+      it('should get entries and clear active cart', () => {
+        service['activeCartId$'] = of('cartId');
+        spyOn(service, 'getEntries').and.returnValue(of([mockCartEntry]));
+
+        service.clearActiveCart();
+        expect(service['getEntries']).toHaveBeenCalled();
```

Comment:
Let's see together if it's possible to improve the coverage on clearActiveCart()